packages/venia-concept/generateGraphQlFragmentTypes.js

```diff
@@ -0,0 +1,43 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+// Right now I've hardcoded this url because we only have process.env.MAGENTO_BACKEND_URL on run time
```

Comment:
If this is done at build time I'd like for it to live in `buildpack`.
We have some basic utilities there already for communicating with the GraphQL backend.

Reply:
Yeah i guess that makes sense, I'm not too experienced with this part of Studio but i'll try to make the change.